PriceCalculater/IDisplayService.cs

```diff
@@ -1,13 +1,14 @@
 ﻿public interface IDisplayService
 {
-    public void Display(String msg,decimal value);
+    public void Display(String msg,decimal value,String currency);
 }
```

Comment:
mmm ,I change on interface ,  I broke contract ,,but Display service is the only class implemnting the interface and dont have problem with that